lib/src/encoder/extensions.dart

```diff
@@ -0,0 +1,59 @@
+part of 'encoder.dart';
+
+extension _AccountsIterableExt on Iterable<AccountMeta> {
```

Comment:
It doesn't make sense to name a private extension, it can just be `extension on Iterable<AccountMeta>`.

Same for other extensions.